sys-process/oomd/oomd-0.5.0-r1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="oomd is an userspace Out-Of-Memory (OOM) killer for Linux systems."
+HOMEPAGE="https://github.com/facebookincubator/oomd"
+SRC_URI="https://github.com/facebookincubator/oomd/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="systemd"
+DEPEND="dev-libs/jsoncpp:=
+	systemd? ( sys-apps/systemd )"
+RDEPEND="${DEPEND}"
+
+CONFIG_CHECK="~CONFIG_PSI"
+ERROR_PSI="CONFIG_PSI is required for oomd to function"
+
+src_configure() {
+	local emesonargs=(
+		$(meson_feature man)
+		$(meson_feature systemd libsystemd)
+	)
+	meson_src_configure
+}
+
+src_install() {
+	meson_src_install
+	einstalldocs
+}
+
+pkg_postinst() {
+	elog "See the following wiki for configuring oomd:"
+	elog "https://github.com/facebookincubator/oomd/blob/master/docs/configuration.md"
+	elog
+	elog "The host must be running cgroup2 alone."
```

Comment:
Wonder if there's a kernel config check related to this too.

Reply:
Good catch!